.travis.yml

```diff
@@ -38,23 +47,22 @@ cache:
 addons:
   apt:
     packages:
-      - x11-apps
-      - xserver-xephyr
-      - xvfb
+    - x11-apps
+    - xserver-xephyr
+    - xvfb
+    - graphviz # for docs
 
 before_install:
-# Setup XvFB
+  # Setup XvFB
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 
 install:
-# Installl requirements
-  - pip install xcffib
-  - pip install -r requirements-dev.txt
-  - if [[ ${EVENTLOOP} == "trollius" ]]; then pip install trollius; fi
-  - if [[ ${EVENTLOOP} == "tulip" ]]; then pip install asyncio; fi
-# build pangocffi module
-  - python -c 'import cffi, sys; sys.exit(cffi.__version_info__[0])' || python libqtile/ffi_build.py
+  - pip install tox python-coveralls
+
+script:
+  - tox
+
 
 after_success:
   - coveralls
```

Comment:
Actually, it's not reporting coverage information at all

https://coveralls.io/builds/4327157